src/member.cc

```diff
@@ -127,11 +127,47 @@ class RedisChainModule {
     }
   }
 
+  enum class MasterMode : int {
```

Comment:
Group with the enum at the top?
http://google.github.io/styleguide/cppguide.html#Declaration_Order